UnityProject/Assets/Scripts/Tool/Pickaxe.cs

```diff
@@ -50,7 +51,8 @@ private void FinishMine(PositionalHandApply interaction)
 		}
 		else
 		{
-			SoundManager.PlayNetworkedAtPos("BreakStone", interaction.WorldPositionTarget, sourceObj: interaction.Performer);
+			// JESTER
+			//SoundManager.PlayNetworkedAtPos("BreakStone", interaction.WorldPositionTarget, sourceObj: interaction.Performer);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)